### DISCUSSION TITLE
lede雷神加速器插件延迟很高

### DISCUSSION THREAD

**Discussion by HOT-POT1** [coolsnowwolf/lede#12384] [Bugs / 问题反馈]
就拿绝地求生来说，win10客户端就50左右延迟，路由器插件直接120左右
![photo_2024-08-05_10-12-37](https://github.com/user-attachments/assets/2e255582-dc0a-4a18-baaf-9155e47cb834)

**Comment by HOT-POT1** [comment]
路由器是京东云百里ax6000，cpu mt7986

**Reply by yanxin152133** [reply]
哪个插件

**Reply by miaoermua** [reply]
> 哪个插件

luci-app-leigod-acc

**Reply by yanxin152133** [reply]
> luci-app-leigod-acc

三克油
